collect_app/src/test/java/org/odk/collect/android/utilities/HtmlUtilsTest.java

```diff
@@ -31,15 +31,13 @@ public void markDownToHtmlEscapesBackslash() {
                 {"_A\\_B\\_C_", "<em>A_B_C</em>"},
                 {"A_B\\_C", "A_B_C"},
                 {"A\\_B_C", "A_B_C"},
-                {"A_B_C", "A<em>B</em>C"},
```

Comment:
I think it would be helpful to keep the existing test cases with the new behavior so it's easier to see what changed. The new cases you added are helpful too so I suggest keeping both.

Reply:
Fixed.